packages/f-checkout/test-utils/component-objects/f-checkout.component.js

```diff
@@ -67,31 +83,41 @@ exports.populateCheckoutForm = addressInfo => {
     fields.addressLine2.input().setValue(addressInfo.line2);
     fields.addressCity.input().setValue(addressInfo.city);
     fields.addressPostcode.input().setValue(addressInfo.postcode);
+    fields.userNote.input().setValue(addressInfo.note);
 };
 
 /**
  * @description
- * Sets the value of the fulfillment time dropdown based on visible text.
- *
- * @param {String} fulfillmentTimeText The text visible text value of the fulfillment time
+ * Sets the value of the order time in dropdown based on visible text.
+ * 
+ * @param {String} orderTime The visible text value of the order time
  */
-exports.selectFulfillmentTime = fulfillmentTimeText => {
-    fulfillmentTimeDropdown().selectByVisibleText(fulfillmentTimeText);
+exports.selectOrderTime = orderTime => {
+    orderTimeDropdown().selectByVisibleText(orderTime);
+};
+/** 
+ * @description
+ * The time of the order should increase when a higher index is applied.
+ * 
+ * @param {Number} index The index of the `orderTimeDropdownOptions` array
+ */
+exports.getOrderTimeOptionText = (index) => {
+    return orderTimeDropdownOptions()[index].getText();
 };
-
 /**
  * @description
- * Sets the value of the user note.
- *
- * @param {String} userNote The user note value to be entered
+ * Sets the value of the user note and grabs the length of characters.
+ * 
+ * @param {Object} addressInfo
+ * @param {String} addressInfo.note The user's extra note
  */
-exports.inputUserNote = userNote => {
-    userNoteInput().setValue(userNote);
+exports.getUserNoteLength = addressInfo => {
+    fields.userNote.input().setValue(addressInfo.note);;
+    return userNoteInput().getValue().length;
 };
-
 /**
  * @description
- * Submits the checkout form.
+ * Clicks 'Go to payment' to submit the form.
```

Comment:
I think this JSDoc should remove the reference to 'Go to payment' as the copy could always get updated in future.

I think 'Submits the checkout form' is sufficient